packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1753,6 +1753,7 @@
 	"workflowSettings.timeoutAfter": "Timeout After",
 	"workflowSettings.timeoutWorkflow": "Timeout Workflow",
 	"workflowSettings.timezone": "Timezone",
+	"workflowHistory.title": "Version History",
```

Comment:
Not sure where the spec is or what it says but I'd expect `Workflow History` for consistency.

Reply:
I saw workflow history everywhere except on the design so I thought I'd add what's on the design but it's rather a product decision, it's easy to change a translation.